### ISSUE TITLE
Test Kitchen fails to connect to Windows 2008 R2 until Enable-PSRemoting is manually executed

### ISSUE THREAD

**Issue by scolligan** [test-kitchen/kitchen-azurerm#77]
** Adding this issue per convo with @stuartpreston on Chef Slack Channel **
** https://chefcommunity.slack.com/archives/C5GU3P9AM/p1521330357000031 **

I've been trying to test against a Windows 2008 R2 instance on Azure and found that it fails to connect with the following error:

```>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Failed to complete #converge action: [A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. - A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. - connect(2) for "13.84.185.74" port 5985 (13.84.185.74:5985)] on default-windows2008
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration```

Once I manually executed `Enable-PSRemoting` on the server, I was able to complete the converge process.  Stuart says there should be code injected into the node that takes care of this to properly prepare it for test kitchen.

Thanks,

Sean

**Comment by cheeseplus** [comment]
As a note we do this in kitchen-ec2 https://github.com/test-kitchen/kitchen-ec2/blob/master/lib/kitchen/driver/ec2.rb#L645-649

**Comment by stuartpreston** [comment]
I remembered how we do this now, or certainly used to. The driver exposes a setting `winrm_powershell_script`. This can be overridden from your .kitchen.yml

We used to document this in the readme but we removed it for some reason. Here’s the commit that adds a Windows 2008 R2 example to the readme:

https://github.com/test-kitchen/kitchen-azurerm/commit/1351c44d9574a7e737cc19409d1ffad1f1cd70b2#diff-04c6e90faac2675aa89e2176d2eec7d8

Hope this makes sense.
